app/validators/notation.js

```diff
@@ -0,0 +1,18 @@
+import _ from "npm:lodash"
+
+export default function validateNotation() {
+  return (key, oldValue) => {
+    let checkStandardNotation = _.map(Object.values(oldValue), standard => {
+      if (standard.statementNotation) {
+        return true
+      } else {
+        return false
+      }
+    })
+
+    let hasNotation = _.filter(checkStandardNotation, notation => {
+      return notation === true
+    })
+    return hasNotation.length / checkStandardNotation.length >= 0.3 ? true : "notation needed"
```

Comment:
so, here, you're checking the length of something that should ahve the length as the originalValue -- just use the original value so the logic is clearer.

`return hasNotation.length / oldValue.length >= 0.3 ? true : "notation needed"`

Also, for this, I wouldn't have the ternary at the end of the lijne. MOve it to it's own line and assign that to a variable so the divsion is more clear:

return hasNotation.length / otherVairable

Also, why are you returning a number here?

Reply:
Followed your logic for the first part and have changed things accordingly (dividing by oldValue.length).  

Feeling a bit confused RE setting ternary to variable and returning a number...will connect on twist later

Reply:
I think I misundetsood earlier -- your returning the error message in the ternary. I'd definitely do something like have a variable called, "let isValid = ..." and then later have "return isValid ? true : "my error message""